src/directory/directory.mjs

```diff
@@ -600,11 +644,29 @@ export const directory = {
             {
               isExternal: true,
               route:
-                'https://ui.docs.amplify.aws/[platform]/connected-components/authenticator',
+                'https://ui.docs.amplify.aws/react/connected-components/liveness',
```

Comment:
Noticed this was pointing to authenticator but should have been pointing to liveness